addon/MetadataStore.js

```diff
@@ -489,7 +489,7 @@ MetadataStore.prototype = {
             });
             break;
           default:
-            throw new Error("Fetched unknown image types: {image.type}");
+            throw new Error(`Fetched unknown image types: ${image.type}`);
```

Comment:
Should this have a `Cu.reportError()` as well, or is that getting logged elsewhere?

Also does anybody know where these `Cu.reportError()` get logged so we can audit those errors for other curious looking strings?